Makefile

```diff
@@ -120,11 +120,12 @@ run-tsurud-token: $(TSR_BIN)
 validate-api-spec: install-swagger
 	$(SWAGGER) validate ./docs/reference/api.yaml
 
-test-int:
-	go get -d github.com/tsuru/platforms/...
-	TSURU_INTEGRATION_examplesdir="${GOPATH}/src/github.com/tsuru/platforms/examples" \
+test-ci-integration:
+	git clone https://github.com/tsuru/platforms /tmp/platforms
+	TSURU_INTEGRATION_examplesdir="/tmp/platforms/examples" \
 	TSURU_INTEGRATION_enabled=1 TSURU_INTEGRATION_verbose=2 TSURU_INTEGRATION_maxconcurrency=4 \
-	TSURU_INTEGRATION_platforms="python" \
+	TSURU_INTEGRATION_platforms="python,go" \
```

Comment:
@morpheu @gvicentin @crgisch: what do you thing about test the most usable platforms like nodejs and static? the tradeoff is duration of pipeline